docs/merkletree/treetex/main.go

```diff
@@ -295,11 +300,18 @@ func modifyRangeNodeInfo() error {
 		// Set leaves:
 		for i := l; i < r; i++ {
 			id := compact.NewNodeID(0, i)
-			modifyNodeInfo(id, func(n *nodeInfo) { n.rangeIndices = append(n.rangeIndices, ri) })
+			modifyNodeInfo(id, func(n *nodeInfo) {
+				n.incPath = true
```

Comment:
It's not an inclusion path, so maybe add a TODO to fix this?

Reply:
I think this was actually unnecessary, dinged it.